### PR TITLE
E_CLASSROOM-298 [FE/BE] [Admin] Profile Account Page

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-298
### Definition of Done
- [x] Layout is the same with the figma design
- [x] Fields are dynamic
- [x] Clicking on "Edit Profile" button will redirect to "Edit Account Info" page
- [x] Clicking on "Change Password" button redirect to "Change Password" page
- [x] User can upload and change profile picture

## Related PR:
- https://github.com/framgia/sph-classroom-els-be/pull/68
#### Main note
- http://localhost:3003/admin/profile
#### Pages Affected
- Profile 
  > http://localhost:3003/admin/profile
- Profile Edit
  > http://localhost:3003/admin/profile/edit
- Edit Password
  > http://localhost:3003/admin/profile/edit-password

### Scenarios/ Test Cases
- [x] Layout is the same with the figma design
- [x] Clicking on "Edit Profile" button will redirect to "Edit Account Info" page
- [x] Clicking on "Change Password" button redirect to "Change Password" page
- [x] User can upload and change profile picture
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
adminprofile2

### CODE REVIEW

**Comment by catampongan** [comment]
## Working
- [x] User can edit profile picture

## Comment
- [x] Buttons are not clickable both `Change password` and `Edit`
- [x] Button in modal should be the same on default button
- [x] Align modal on profile edit
-----------------------
Screenshot 2022-03-08 180836
bugfix-1
-------------------------
- Total Test Time: 0.25 hr

**Comment by catampongan** [comment]
## Approved PR #134 
- [x]  User can edit profile picture
- [x] Able to redirect change password page when `Change password` button is clicked
- [x] Able to redirect edit info page when `Edit` button is clicked
- [x] Design is the same in Figma
- [x] Layout is dynamic
-------------------
image
task-298
---------------
- Total Test Time: 0.5 hr
- Total Iterations: 1
- Total Comments: 3

File: src/pages/admin/profile/ProfileView/index.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        setAvatar(data.avatar);
+        setStatus(false);
+        setModalShow(false);
+      });
+    } catch (error) {
+      if (error?.response?.data?.errors) {
+        toast('Error', 'Please enter a valid input to successfully upload.');
+        setError(error?.response?.data?.errors);}
+      setSubmitStatus(false);
+    }
```
Comment (karlolee50) [inline src/pages/admin/profile/ProfileView/index.js]: use .then and .catch

File: src/pages/admin/profile/ProfileView/index.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                  <Form.Control
+                    style={{ fontSize: '14px' }}
+                    value={profileName?.email}
+                    disabled="disabled"
+                    className={style.formControlstyle}
+                  />
+                </Form.Group>
+                <a href="/admin/profile/edit-password">
+                  <Button className={style.changepassbutton} variant="primary">
+                Change Password
```
Comment (karlolee50) [inline src/pages/admin/profile/ProfileView/index.js]: fix indentation

File: src/pages/admin/profile/ProfileView/index.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                    value={profileName?.name}
+                    disabled="disabled"
+                    className={style.formControlstyle}
+                  />
+                </Form.Group>
+
+                <Form.Group className={style.marginForForm} controlId="formBasicEmail">
+                  <Form.Label
+                    className={style.FormGroupStyle}>
+                Email
```
Comment (karlolee50) [inline src/pages/admin/profile/ProfileView/index.js]: fix indentation

File: src/pages/admin/profile/ProfileView/index.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                  />
+                </Form.Group>
+                <a href="/admin/profile/edit-password">
+                  <Button className={style.changepassbutton} variant="primary">
+                Change Password
+                  </Button>
+                </a>
+                <a href="/admin/profile/edit">
+                  <Button className={style.editbutton} variant="primary">
+                Edit
```
Comment (karlolee50) [inline src/pages/admin/profile/ProfileView/index.js]: fix indentation

File: src/pages/admin/profile/ProfileView/index.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            {profileName === null ? (
+              <div className={style.loading}>
+                <Spinner animation="border" role="status"></Spinner>
+                <span className={style.loadingWord}>Loading</span>
+              </div>
+            ) : (
+              <Form style={{ marginTop: '20px' }}>
+                <Form.Group className={style.marginForForm} controlId="formBasicName">
+                  <Form.Label className={style.FormGroupStyle}>
+                Name
```
Comment (karlolee50) [inline src/pages/admin/profile/ProfileView/index.js]: fix indentation

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**